.ci/vfio_jenkins_job_build.sh

```diff
@@ -181,7 +181,7 @@ create_config_iso() {
 pull_fedora_cloud_image() {
 	fedora_img="$1"
 	fedora_img_cache="${fedora_img}.cache"
-	fedora_version=32
+	fedora_version=34
```

Comment:
Could we bump this to F35 now that's been released?

Reply:
it's not stable yet https://github.com/kata-containers/kata-containers/pull/2962/files, isn't it?

Reply:
Good catch! Thanks @devimc.